### PR TITLE
User Profile Card

### PR DESCRIPTION
**Pull request by BeckyRushton**
Built staff card component and added styling to match example.

### CODE REVIEW

File: src/components/UserProfileCard/UserProfileCard.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./UserProfileCard.scss";
+import staffPicture from "../../assets/images/users/staff-picture.png";
+
+const UserProfileCard = () => {
```
Comment (TristanPere) [inline src/components/UserProfileCard/UserProfileCard.jsx:5]: Currently takes no prompts so can't be changed when called. Would be great to take a staff name, type of employment, and image as prompts as well as any others you deem necessary

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import Layout from "../../components/Layout/Layout";
+import UserProfileCard from "../../components/UserProfileCard/UserProfileCard";
```
Comment (TristanPere) [inline src/pages/Home/Home.jsx:2]: Although you're working on it and it needs to be rendered to work on, when making a pull request only include items that need to be rendered at that time and place. I will go over this in the E.O.D. chat.

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -12,6 +13,7 @@ const Home = () => {
         maxime tempora, temporibus beatae voluptas repellat rerum. Dignissimos,
         necessitatibus.
       </p>
+      <UserProfileCard />
```
Comment (TristanPere) [inline src/pages/Home/Home.jsx:16]: Same comment as above

**Review by TristanPere — CHANGES_REQUESTED** [review]
Component is looking great, just like the one from figma! It needs some props so it can be modular dependent on given details (props). Also note points on not-yet-rendered components.

**Review by olirob93 — APPROVED** [review]

**Review by TristanPere — APPROVED** [review]

**Pull request merged**